packages/frontend-2/components/projects/DashboardDiscoverableWorkspaces.vue

```diff
@@ -0,0 +1,74 @@
+<!--
+  ___       __   ________  ___  _________
+|\  \     |\  \|\   __  \|\  \|\___   ___\
+\ \  \    \ \  \ \  \|\  \ \  \|___ \  \_|
+ \ \  \  __\ \  \ \   __  \ \  \   \ \  \
+  \ \  \|\__\_\  \ \  \ \  \ \  \   \ \  \
+   \ \____________\ \__\ \__\ \__\   \ \__\
+    \|____________|\|__|\|__|\|__|    \|__|
+
+This file is only meant to be used in its new shiny
+home on the workspaces dashboard page! It is living
+here behind a flag for testing reasons.
+-->
+
+<template>
+  <div class="w-full flex flex-col gap-4">
+    <div
+      v-for="workspace in discoverableWorkspaces"
+      :key="workspace.id"
+      class="w-full p-4 flex flex-row items-center justify-between border border-outline-3 border-primary rounded-xl bg-foundation"
+    >
+      <div class="flex flex-row flex-shrink items-center">
+        <div class="w-8 h-8 mr-4 rounded-full bg-primary"></div>
+        <div class="flex flex-col justify-start">
+          <div class="mb-1">
```

Comment:
You can use `<p class="text-body-xs text-foreground">` to align with the text styles, and semantically it's better to use a `<p>` tag instead of a `<div>`